test/metadataSpec.js

```diff
@@ -25,6 +28,7 @@ describe('Metadata', function () {
             currency_code: null,
             currency_symbol: null,
             public_key: '9PAqTUEptSeQCOp/0FQTm3rkFnUFaYEUEwCcyyySQP0=',
```

Comment:
Since this is going to be a breaking change, I think maybe we should change this key to be condition_public_key or ed25519_public_key. It's otherwise confusing to have one generic and one that's specific